lib/flipper-file/flipper-file-string.c

```diff
@@ -0,0 +1,66 @@
+#include <furi.h>
+#include "flipper-file.h"
+#include "flipper-file-i.h"
+#include "flipper-file-tool.h"
+
+static bool flipper_file_write_string_internal(
+    File* file,
+    const char* key,
+    const void* data,
+    const uint16_t data_size) {
+    bool result = false;
+    (void)data_size;
+
+    do {
+        result = flipper_file_write_key(file, key);
+        if(!result) break;
+
+        result = file_tool_write(file, string_get_cstr(data), string_size(data));
+        if(!result) break;
+
+        result = file_tool_write_eol(file);
+    } while(false);
+
+    return result;
+};
+
+bool flipper_file_read_string(FlipperFile* flipper_file, const char* key, string_t data) {
+    furi_assert(flipper_file);
+
+    bool result = false;
+    if(flipper_file_seek_to_key(flipper_file->file, key)) {
+        if(file_tool_read_line(flipper_file->file, data)) {
+            result = true;
+        }
+    }
+    return result;
+}
+
+bool flipper_file_write_string(FlipperFile* flipper_file, const char* key, string_t data) {
+    furi_assert(flipper_file);
+    return flipper_file_write_string_internal(flipper_file->file, key, data, 0);
+}
+
+bool flipper_file_write_string_cstr(FlipperFile* flipper_file, const char* key, const char* data) {
+    bool result = false;
```

Comment:
Asserting one places and ignoring another

Reply:
flipper_file_write_string_cstr calls flipper_file_write_string, which already contains assert